src/template.js

```diff
@@ -0,0 +1,64 @@
+var globby = require('globby');
+var path  = require('path');
+var utils = require('./utils');
+
+/**
+ * Register helpers on the passed Handlebars instance.
+ * Accept an object with helperKey => helperFunctions,
+ * or a glob (as Array or string) of files (modules) to
+ * register. In the latter case, the filename w/o extension
+ * will be used as the helper key.
+ *
+ * @param {Object} Handlebars handlebars instance
+ * @param {Object} || [{Array} || {String} glob]
+ * @param return {Promise}, resolving to
+ *           {Object} of all helpers registered on Handlebars
+ */
+function prepareHelpers (Handlebars, helpers = {}) {
+  if (typeof helpers === 'string' || Array.isArray(helpers)) {
+    return globby(helpers).then(helperPaths => {
+      helperPaths.forEach(helperPath => {
+        const helperKey = path.basename(helperPath, path.extname(helperPath));
+        Handlebars.registerHelper(helperKey, require(helperPath));
```

Comment:
General question: What are the pros/cons of using `require` instead of `fs` for something like this?